finish/src/main/webapp/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,3 @@
+# tag::config[]
+io.openliberty.guides.inventory.client.SystemClient/mp-rest/url=http://localhost:9080/system
```

Comment:
if you really dont need the `defaultRestClient`, you probably dont need this config file.